### ISSUE TITLE
the final series of matches on live are interrupted every 55 minutes

### ISSUE THREAD

**Issue by nickomaru** [maxgalbu/xbmc.plugin.video.nba#16]
Kodi create new link
http://nlds130.cdnak.neulion.com/nlds/nba/..........

Dexpires%253D1465864219
Dexpires%253D1465866213
Dexpires%253D1465866744
Dexpires%253D1465869802

**Comment by nzeppos** [comment]
I don't think this is just the finals, as I've been experiencing such issues for practically all the live games I watched during the playoffs. As a workaround I always stop and restart the stream at the beginning of the fourth quarter, so as to avoid having any interruptions while the last minutes are on.

It doesn't always stop at exactly the 55th minute, to me it can happen at anytime after the 50th minute (yesterday it stopped after 53' and then again after 59')..

**Comment by nickomaru** [comment]
Yes, it is, but this option does not suit me

**Comment by maxgalbu** [comment]
Yes this is common with all the live videos, they have an expiration time and i need to request new ones as the video plays. I think it's feasible with a service plugin (a plugin running in the background) that changes the url after some time (eg https://github.com/tamland/xbmc-addon-watchdog/blob/master/service.py). However I don't expect it to work seamlessly like the website (assuming it's possible)

**Comment by besarbb** [comment]
Hello maxgalbu. Firstly thanks a lot for the add-on.

But I am having real difficulties watching any live games to be honest. It simply stops as reported above, and works fine as soon as it is restarted. Is there anything that can be done about this?

Thank you :)

**Comment by maxgalbu** [comment]
@besarbb 

> I think it's feasible with a service plugin (a plugin running in the background) that changes the url after some time (eg https://github.com/tamland/xbmc-addon-watchdog/blob/master/service.py). However I don't expect it to work seamlessly like the website (assuming it's possible)

I have a possible fix for the 0.9 version, just need some time to fix it.
